app.js

```diff
@@ -36,45 +9,91 @@ function errorMessage(message) {
   }, 4000);
 }
 
-//  Add book
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
 
-function addBook(title, author) {
-  const id = Date.now();
-  const bookList = {
-    id,
-    title,
-    author,
-  };
+  addBook() {
+    const { title, author, id } = this;
+    const bookList = { title, author, id };
+    books = JSON.parse(localStorage.getItem('books'));
+    if (title === '' || author === '') {
+      errorMessage('Kindly fill all inputs');
+    } else if (books !== null) {
+      books.push(bookList);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+    } else {
+      books = [];
+      books.push(bookList);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+  }
+
+  remove() {
+    const { id } = this;
+    books = books.filter((book) => {
+      if (book.id === id) {
+        return false;
+      }
+      return true;
+    });
 
-  if (title === '' || author === '') {
-    errorMessage('Kindly fill all inputs');
-  } else {
-    books.push(bookList);
     localStorage.setItem('books', JSON.stringify(books));
-    displayBooks(bookList.id, bookList.title, bookList.author);
   }
 }
 
-// Global variable to check local storage
+// const book = new Book();
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments. 😉 💯

Reply:
Thank you @BenMukebo for your feedback. I will work on the author that has  not been added to this particular branch, but for the "getting started" section, I had already modified it to suit my project, I don't think it should be listed directly as it was in the template( moreover, the template is a guide and not to copy and paste)
 secondly, for the MIT license, It is included already, I don't know if it is an oversight from your end.

I will remove the redundant code, work on the second author, and revert. Thanks

Reply:
Okay, Thank you so much for the review.